packages/babel-plugin-optimize-react/package.json

```diff
@@ -15,6 +15,7 @@
     "@babel/core": "^7.1.0"
   },
   "devDependencies": {
+    "@babel/preset-env": "7.2.3",
```

Comment:
Is it necessary to used babel?

Reply:
This preset is only used to test how this plugin works with other plugins.